wcomponents-test-lib/src/main/java/com/github/bordertech/wcomponents/test/selenium/element/SeleniumWTabSetWebElement.java

```diff
@@ -1,12 +1,11 @@
 package com.github.bordertech.wcomponents.test.selenium.element;
 
+import java.util.List;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
-import java.util.List;
-
 /**
  * The Wcomponent WTabSet web element.
  */
```

Comment:
I feel the starting point of this extension is flawed as it does not make any attempt to test if the element obtained from the call to `super(element, driver);` is, in fact, a WTabSet. Could you please beef this up if you are going to make use of this extension.